### PR TITLE
Shift to yarn

### PR DESCRIPTION
**Pull request by aquibbaig**
While the other pull request was pending, I decided to create a PR cause we need to move this issue sooner.

### CODE REVIEW

**Comment by Harkishen-Singh** [comment]
Sure. This issue is out for long. We can get this merged anytime soon as the CI passes.

**Comment by aquibbaig** [comment]
@Harkishen-Singh Squash and merge.

File: local-config.yml [hunk 1]
```diff
@@ -46,11 +46,11 @@ routes:
...
+    mean: null
   services-state:
-    ping: passive
-    flood-ping: passive
-    jitter: passive
-    req-res-delay-or-monitoring: passive
+    ping: active
+    flood-ping: active
+    jitter: active
+    req-res-delay-or-monitoring: active
```
Comment (Harkishen-Singh) [inline local-config.yml:56]: please change these to `passive`

File: local-config.yml [hunk 2]
```diff
@@ -46,11 +46,11 @@ routes:
   - name: ei
     value: UTF-8
 utils:
-  monitor-length:
-    mode: 0
-    mean: 0
+  response-length:
```
Comment (Harkishen-Singh) [inline local-config.yml:49]: please ensure that this is correct.

**Pull request merged**